wandb/sdk/internal/job_builder.py

```diff
@@ -215,7 +218,14 @@ def _build_image_job(
     ) -> Tuple[Artifact, ImageSourceDict]:
         image_name = metadata.get("docker")
         assert isinstance(image_name, str)
-        name = make_artifact_name_safe(f"job-{image_name}")
+
+        raw_image_name = image_name
+        if ":" in image_name:
+            raw_image_name, tag = image_name.split(":")
+            # TODO(gst): make this better
+            self.aliases += [tag]
```

Comment:
this is terrible, we instead want to communicate this to a backend process. we can put this in the metadata, and at log-time, peek into it and see if we have any tags?